k8s/stage-analytics-frontend.yaml

```diff
@@ -0,0 +1,41 @@
+###########################################################
```

Comment:
Same here, we could remove the analytics configs

Reply:
Also resolved